apps/src/utils.js

```diff
@@ -128,6 +131,7 @@ Function.prototype.inherits = function (parent) {
  * done so that level builders can specify required blocks with wildcard fields.
  */
 export function wrapNumberValidatorsForLevelBuilder() {
+  declare var Blockly: Object;
```

Comment:
yeah, it would be better to put this declaration at the top of the file. Not sure what the deal is with the documentation. That is kind of confusing.